src/Stepper/Step.js

```diff
@@ -0,0 +1,180 @@
+      step: {
+        width: `${width}%`,
+        display: 'table-cell',
+        position: 'relative',
+        paddingTop: circleTop,
+      },
+
+      circle: {
+        width: size,
+        height: size,
+        margin: '0 auto',
+        backgroundColor: 'transparent',
+        borderRadius: '50%',
+        borderWidth: 3,
+        borderStyle: 'solid',
+        borderColor: '#fff',
+        textAlign: 'center',
+        padding:1,
+        fontSize: circleFontSize,
+        color: circleFontColor,
+        display: 'block',
+        opacity: defaultOpacity,
+
+      },
+      activeCircle: {
+        backgroundColor: activeColor,
+        opacity: activeOpacity
+      },
+      completedCircle: {
+        backgroundColor: completeColor,
+        opacity: completeOpactiy,
+      },
+      index: {
+        lineHeight: `${size + circleFontSize / 4}px`,
+        color: circleFontColor
+      },
+      title: {
+        marginTop: titleTop,
+        fontSize: titleFontSize,
+        fontWeight: '300',
+        textAlign: 'center',
+        display: 'block',
+        color: defaultTitleColor,
+        opacity: defaultOpacity
+      },
+      activeTitle: {
+        color: activeTitleColor,
+        opacity: activeOpacity
+      },
+      completedTitle: {
+        color: completeTitleColor,
+        opacity: completeOpactiy
+      },
+      leftBar: {
+        position: 'absolute',
+        top: circleTop + size / 2,
+        height: 1,
+        borderTopStyle: 'dashed',
+        borderTopWidth: 1,
+        borderTopColor: defaultColor,
+        left: 0,
+        right: '50%',
+        marginRight: size / 2 + 5,
+        opacity: defaultOpacity
+      },
+      rightBar: {
+        position: 'absolute',
+        top: circleTop + size / 2,
+        height: 1,
+        borderTopStyle: 'dashed',
+        borderTopWidth: 1,
+        borderTopColor: defaultColor,
+        right: 0,
+        left: '50%',
+        marginLeft: size / 2 + 5,
+        opacity: defaultOpacity
+      },
+      completedBar: {
+        borderTopStyle: 'dashed',
+        borderTopWidth: 1,
+        borderTopColor: completeColor,
+        opacity: completeOpactiy
+      },
+    };
+  }
+
+  render() {
+    const { title, index, active, completed, first, isLast, href, onClick } = this.props;
+
+    const styles = this.getStyles();
+    const circleStyle = Object.assign(
+      styles.circle,
+      completed ? styles.completedCircle : {},
+      active ? styles.activeCircle : {},
+    );
+
+    const titleStyle = Object.assign(
+      styles.title,
+      completed ? styles.completedTitle : {},
+      active ? styles.activeTitle : {},
+    );
+    const leftStyle = Object.assign(styles.leftBar, (active || completed) ? styles.completedBar : {});
+    const rightStyle = Object.assign(styles.rightBar, completed ? styles.completedBar : {});
+
+    return (
+      <div style={ styles.step }>
+          <div style={ circleStyle }>
+            {active || completed ? (
```

Comment:
The majority of this code came from [react-stepper](https://github.com/mu29/react-stepper). All I added were the borders, opacity changes, and optionally dashed bars. Many of the values in here are hardcoded for our specific example inside the demo-app, but I have been working on adding completely reusable versions of these features to the actual react-stepper library. I've only gotten a couple pull requests accepted so far and am still working to finish the circle borders, but I'd like to think we can use the actual repository for these features in the end. I can try to make these styling changes and submit them as a PR to the react-stepper repo or I can make these changes on the demo-application only and we can just make use of the Stepper component with hardcoded values that we are already using. Let me know which you would prefer.